source/index.html.md

```diff
@@ -798,6 +902,88 @@ Only one 3-D Secure version can be used for a given authorization.
   </dd>
 </dl>
 
+##### Scheme reference to series
+
+If the previous-in-series authorization was made via this API, you should use
+`series[previous][id]` to reference the previous-in-series authorization.
+Otherwise, you must obtain explicit approval from Clearhaus to use the raw
+scheme values grouped in <code>series[previous][mastercard]</code> and
+<code>series[previous][visa]</code>.
+
+The Mastercard specific reference to the series contains the following parts.
+
+<dl class="dl-vertical">
+  <dt>
+    series[previous][mastercard][tid]
+    <span class="type">[A-Za-z0-9]{3}[A-Za-z0-9]{6}[0-9]{4}[ ]{2}</span>
+  </dt>
+  <dd>
+    Trace ID being the concatenation of values
+    Data element 63 subfield 1 (Financial Network Code) (position 1-3),
+    Data element 63 subfield 2 (Banknet Reference Number) (position 4-9),
+    Data element 15 (Date, Settlement) (position 10-13), and
+    two spaces;
+    to be used in Data Element 48, Subfield 63.
+
+    <div class="type">Conditional.
+    Required if <code>series[previous][mastercard]</code> is present.
+    Cannot be present if <code>series[previous][id]</code> or
+    <code>series[previous][visa]</code> is present.</div>
+  </dd>
+
+  <dt>
+    series[previous][mastercard][exemption]
+    <span class="type">(<code>fixed_amount_series</code>|<code>variable_amount_series</code>)</span>
+  </dt>
+  <dd>
+    The <i>Mastercard exemption</i> to be used for the series indicating if the
+    series is a fixed-amount or a variable-amount series.
+    <ul>
+      <li><code>fixed_amount_series</code>:
+        The series is a fixed-amount series. (MPMI value <code>03</code>.)
+      </li>
+      <li><code>variable_amount_series</code>:
+        The series is a variable-amount series. (MPMI value <code>01</code>.)
+        A <i>Mastercard exemption</i> (not formally an acquirer exemption for
+        SCA) indicating that the transaction is out of scope for SCA.
```

Comment:
Uhm, but MIT **is** "formally" a Mastercard acquirer exemption for SCA?

Reply:
It is not for Visa.

Reply:
Well, Mastercard does not mention MPMI = `01` as an exemption, but as an indication that it's out of scope — but they're also using the word "exemption" when talking about the MPMI field itself 🤷‍♂️

I'm understanding it as if they're being veeery slippery and working around PSD2 RTS regarding the requirement for SCA for amount adjustments and fixed amount recurring (and the internal conflict in the RTS that payment instruments may not be discriminated) 😃
Visa took it by the horns and kinda just allows themselves to disagree with the RTS.

I am definitely nowhere near quadratic on how we should word this. I am very open for suggestions 🙂

Reply:
I was surprised to read about MPMI values here. Are they really necessary? Does the audience need to know? I haven't (yet) read about them anywhere else, and I think I've read a fair amount of PSD2 SCA documents. If we zoom out, a number of SCA acquirer exemptions exist, and some of them are shared by both schemes, and some of them are only for one of them. AFAICS, MIT is an exemption only for Mastercard. Visa just says MIT is out of scope rather than define an exemption. In this context I don't understand the parenthesis in "A Mastercard exemption (not formally an acquirer exemption for SCA) [...]", and my reaction was actually only to related to that.

But it's certainly a minefield :slightly_smiling_face:

Reply:
Shortly regarding MPMI: We haven't thought about this issue before, it was only now I realized that the consequences of a subsequent-in-series being marked with MPMI `01` or `03`.

If a previous-in-series created via another acquirer was a first-in-series, then the PSP should choose if the current subsequent-in-series (the first subsequent in series, because the previous was a first-in-series) should allow for varying amount or not.

If a previous-in-series created via another acquirer was a subsequent-in-series, then the PSP should make the MPMI match the one used for the previous-in-series. Otherwise, the series change between being a varying-amount and fixed-amount series.

Reply:
Argh. Indeed not simple, but it makes good sense then. I assume this is or will be a part of our separate guide.